docs/index.bs

```diff
@@ -3127,6 +3129,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
           1. [=Abort a running script|Abort the script=] currently running in |serviceWorker|.
           1. Set |serviceWorker|'s [=start status=] to null.
+          1. Unset |serviceWorker|'s [=ServiceWorkerGlobalScope is ready=].
```

Comment:
Can we also address this comment?
https://github.com/w3c/ServiceWorker/pull/1701/files#r1484868955

Reply:
Partially addressed the comment by adding " flag".  However, I believe the main topic of the comment is renaming.
I am considering to call it "ServiceWorkerGlobalScope CSP ready flag"
What do you think?